### ISSUE TITLE
Update deprecated tensorflos & keras function calls

### ISSUE THREAD

**Issue by florian-huber** [NLeSC/mcfly#196]
I recently got a bunch of deprecated warnings which we maybe need to look into...

WARNING: Logging before flag parsing goes to stderr.
W0722 14:02:46.179984 12396 deprecation_wrapper.py:119] From C:\Users\FlorianHuber\Anaconda3\envs\mne\lib\site-packages\keras\backend\tensorflow_backend.py:74: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0722 14:02:46.211227 12396 deprecation_wrapper.py:119] From C:\Users\FlorianHuber\Anaconda3\envs\mne\lib\site-packages\keras\backend\tensorflow_backend.py:517: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

W0722 14:02:46.258124 12396 deprecation_wrapper.py:119] From C:\Users\FlorianHuber\Anaconda3\envs\mne\lib\site-packages\keras\backend\tensorflow_backend.py:133: The name tf.placeholder_with_default is deprecated. Please use tf.compat.v1.placeholder_with_default instead.

W0722 14:02:46.289368 12396 deprecation_wrapper.py:119] From C:\Users\FlorianHuber\Anaconda3\envs\mne\lib\site-packages\keras\backend\tensorflow_backend.py:4138: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

W0722 14:02:46.320579 12396 deprecation_wrapper.py:119] From C:\Users\FlorianHuber\Anaconda3\envs\mne\lib\site-packages\keras\backend\tensorflow_backend.py:174: The name tf.get_default_session is deprecated. Please use tf.compat.v1.get_default_session instead.

W0722 14:02:46.367473 12396 deprecation_wrapper.py:119] From C:\Users\FlorianHuber\Anaconda3\envs\mne\lib\site-packages\keras\backend\tensorflow_backend.py:1834: The name tf.nn.fused_batch_norm is deprecated. Please use tf.compat.v1.nn.fused_batch_norm instead.

W0722 14:02:47.445311 12396 deprecation.py:506] From C:\Users\FlorianHuber\Anaconda3\envs\mne\lib\site-packages\keras\backend\tensorflow_backend.py:3445: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
W0722 14:02:47.492175 12396 deprecation_wrapper.py:119] From C:\Users\FlorianHuber\Anaconda3\envs\mne\lib\site-packages\keras\optimizers.py:790: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.
